docs/drivers/is31fl3761.md

```diff
@@ -0,0 +1,284 @@
+
+To configure this, set the `IS31FL3761_I2C_ADDRESS_n` defines to one of the following in your `config.h`, where *n* denotes the driver index:
+
+|Define                          |Value |
+|--------------------------------|------|
+|`IS31FL3761_I2C_ADDRESS_GND_GND`|`0x30`|
+|`IS31FL3761_I2C_ADDRESS_GND_SCL`|`0x31`|
+|`IS31FL3761_I2C_ADDRESS_GND_SDA`|`0x32`|
+|`IS31FL3761_I2C_ADDRESS_GND_VCC`|`0x33`|
+|`IS31FL3761_I2C_ADDRESS_SCL_GND`|`0x34`|
+|`IS31FL3761_I2C_ADDRESS_SCL_SCL`|`0x35`|
+|`IS31FL3761_I2C_ADDRESS_SCL_SDA`|`0x36`|
+|`IS31FL3761_I2C_ADDRESS_SCL_VCC`|`0x37`|
+|`IS31FL3761_I2C_ADDRESS_SDA_GND`|`0x38`|
+|`IS31FL3761_I2C_ADDRESS_SDA_SCL`|`0x39`|
+|`IS31FL3761_I2C_ADDRESS_SDA_SDA`|`0x3A`|
+|`IS31FL3761_I2C_ADDRESS_SDA_VCC`|`0x3B`|
+|`IS31FL3761_I2C_ADDRESS_VCC_GND`|`0x3C`|
+|`IS31FL3761_I2C_ADDRESS_VCC_SCL`|`0x3D`|
+|`IS31FL3761_I2C_ADDRESS_VCC_SDA`|`0x3E`|
+|`IS31FL3761_I2C_ADDRESS_VCC_VCC`|`0x3F`|
+
+### PWM Frequency {#pwm-frequency}
+
+The PWM frequency can be adjusted (for IS31FL3761 only) by adding the following to your `config.h`:
+
+```c
+#define IS31FL3761_PWM_FREQUENCY IS31FL3761_PWM_FREQUENCY_92K_HZ
+```
+
+Valid values are:
+
+|Define                           |Frequency       |
+|---------------------------------|----------------|
+|`IS31FL3761_PWM_FREQUENCY_92K_HZ`|92kHz (default) |
+|`IS31FL3761_PWM_FREQUENCY_46k_HZ`|46kHz           |
+|`IS31FL3761_PWM_FREQUENCY_23k_HZ`|23kHz           |
+|`IS31FL3761_PWM_FREQUENCY_11k5_HZ`|11.5kHz        |
+
+### De-Ghosting {#de-ghosting}
+
+These settings control the pullup and pulldown resistor values on the `CSx` and `SWy` pins respectively, for the purposes of eliminating ghosting. Refer to the datasheet (p. 18) for more information on how and why this occurs.
+
+To adjust the resistor values, add the following to your `config.h`:
+
+```c
+#define IS31FL3761_SW_PULLUP IS31FL3761_PUR_16K_OHM
+#define IS31FL3761_CS_PULLDOWN IS31FL3761_PDR_0K5_OHM
+```
+
+Valid values for `IS31FL3761_SW_PULLUP` are:
+
+|Define                  |Resistance          |
+|------------------------|--------------------|
+|`IS31FL3761_PUR_0_OHM`  |No pull up          |
+|`IS31FL3761_PUR_0K5_OHM`|PVCC-2.3V           |
+|`IS31FL3761_PUR_1K_OHM` |PVCC-2.0V           |
+|`IS31FL3761_PUR_2K_OHM` |PVCC-1.7V           |
+|`IS31FL3761_PUR_4K_OHM` |PVCC-1.4V           |
+|`IS31FL3761_PUR_8K_OHM` |PVCC-1.1V           |
+|`IS31FL3761_PUR_16K_OHM`|PVCC-0.8V (default) |
+|`IS31FL3761_PUR_32K_OHM`|PVCC                |
+
+Valid values for `IS31FL3761_CS_PULLDOWN` are:
+
+|Define                  |Resistance     |
+|------------------------|---------------|
+|`IS31FL3761_PDR_0_OHM`  |No pull up     |
+|`IS31FL3761_PDR_0K5_OHM`|2.6V (default) |
+|`IS31FL3761_PDR_1K_OHM` |2.2V           |
+|`IS31FL3761_PDR_2K_OHM` |1.9V           |
+|`IS31FL3761_PDR_4K_OHM` |1.6V           |
+|`IS31FL3761_PDR_8K_OHM` |1.2V           |
+|`IS31FL3761_PDR_16K_OHM`|0.9V           |
+|`IS31FL3761_PDR_32K_OHM`|GND            |
```

Comment:
The names of these defines look really wrong (`0_OHM` looks like a short, but actually that value means “no connection” here).